velox/vector/FunctionVector.h

```diff
@@ -55,6 +52,16 @@ class Callable {
       const std::vector<VectorPtr>& args,
       const BufferPtr& elementToTopLevelRows,
       VectorPtr* result) = 0;
+
+  /// Same as 'apply', but suppress the eval error.
```

Comment:
nit: Would be nice to clarify that errors are returned in 'elementErrors' and errors in 'context' are not updated.